Makefile

```
@@ -305,35 +229,38 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
-### install_cert_manager: install Cert Mananger v1.0.4 on the cluster
-install_cert_manager:
-	kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.yaml
 
-_operator_sdk:
-	@{ \
-		if ! command -v operator-sdk &> /dev/null; then \
-			echo 'operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but it is not installed' ;\
-			exit 1 ;\
-		else \
-			SDK_VER=$$(operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2) && \
-			if [ "$${SDK_VER}" != $(OPERATOR_SDK_VERSION) ]; then \
-				echo "WARN: operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but $${SDK_VER} found" \
-				echo "WARN: Please use the recommended operator-sdk if you face any issue" \
-			fi \
-		fi \
-	}
-ifneq ($(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2),$(OPERATOR_SDK_VERSION))
-	@echo 'WARN: operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but $(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2) found.'
-	@echo 'WARN: Please use the recommended operator-sdk if you face any issue.'
-endif
-
-_check_cert_manager:
-ifeq ($(PLATFORM),kubernetes)
-	if ! ${K8S_CLI} api-versions | grep -q '^cert-manager.io/v1$$' ; then \
-		echo "Cert-manager is required for deploying on Kubernetes. See 'make install_cert_manager'" ;\
-		exit 1 ;\
+.PHONY: compile-devworkspace-controller
+compile-devworkspace-controller:
+	export ARCH="$(shell uname -m)";
+	if [[ "$(ARCH)" == "x86_64" ]]; then
+		export ARCH="amd64";
+	elif [[ "$(ARCH)" == "aarch64" ]]; then
+		export ARCH="arm64";
 	fi
```

The whole rule should take place on one line or Florent will complain :D (unless these rules are only used within docker builds with a modern version of `make` installed). Something like
```suggestion
	export ARCH="$(shell uname -m)"; \
	if [[ "$(ARCH)" == "x86_64" ]]; then \
		export ARCH="amd64"; \
	elif [[ "$(ARCH)" == "aarch64" ]]; then \
		export ARCH="arm64"; \
	fi ;\
```

This is because macOS by default uses `make` from 15 years ago before `.ONESHELL` was added.

---

:smile: Sorry, didn't realize that - I've updated it!